### PR TITLE
Link WASM example to overview page and improve crud app

### PR DESCRIPTION
**Pull request by prakhargupta1**
- Linked WASM, data provider example to the Overview page
- Added images to examples where it was missing
- Delinked one of the custom data grid app

### CODE REVIEW

**Comment by prakhargupta1** [comment]
@bharatkashyap I have added Stripe app to the examples overview page, can you add the screenshot for it? I don't have the token.

**Comment by bharatkashyap** [comment]
> @bharatkashyap I have added Stripe app to the examples overview page, can you add the screenshot for it? I don't have the token.

I think we can add a README.md (similar to the other apps) to the Stripe example. test it out, polish the experience - and then add it to the examples page.

File: docs/src/modules/components/ExamplesGrid.js [hunk 1]
```diff
@@ -75,6 +76,21 @@ function layouts() {
       href: 'https://github.com/mui/mui-toolpad/tree/master/examples/graphql',
       source: 'https://github.com/mui/mui-toolpad/tree/master/examples/graphql',
     },
+    {
+      title: 'With WASM',
```
Comment (apedroferreira) [inline docs/src/modules/components/ExamplesGrid.js:80]: How common is the WASM usecase?
If we have a limited number of slots for examples, eventually we might want to show just the common ones.
But for now I guess that we should show whatever we have!

**Review by apedroferreira — APPROVED** [review]
All good! Just some minor text changes.

File: docs/src/modules/components/ExamplesGrid.js [hunk 2]
```diff
@@ -55,15 +55,16 @@ function layouts() {
...
+      title: 'Customized data grid',
+      description: 'A Toolpad app that shows how to customize a data grid using code component.',
       src: '/static/toolpad/marketing/datagrid-columns.png',
       href: 'https://github.com/mui/mui-toolpad/tree/master/examples/datagrid-columns',
       source: 'https://github.com/mui/mui-toolpad/tree/master/examples/datagrid-columns',
     },
     {
-      title: 'Custom data grid',
-      description: 'A basic Toolpad app that shows how to add a code component inside a data grid.',
+      title: 'Custom data grid columns',
```
[the remark this answers is not in the document]
Comment (prakhargupta1) [reply docs/src/modules/components/ExamplesGrid.js:65]: Yes, delinked the one with the chips component. It is already available [here](https://mui.com/toolpad/how-to-guides/customize-datagrid/#configuring-columns).

**Pull request merged**